test/ignores/macos.js

```diff
@@ -0,0 +1,49 @@
+const todo = [
```

Comment:
>If the images are visually indistinguishable and dif is suitably low (< 1% for small images and less for larger or detail-oriented ones), raise the "allowed" threshold just above the dif.
>Otherwise, add to a platform-specific ignore file.

Can we improve the [development documentation](https://github.com/mapbox/mapbox-gl-js/tree/main/test#tests) and guidelines on the topic of ignore lists? So that it's clear what is the approach to take for anyone who didn't follow that work, and adding new render test isn't confusing whenever new platform tests fail due to minor differences.

It should at least mention the approach taken above, as well as describe and reference the ignore files used between platform.

Reply:
Good idea!

I do feel a bit nervous about standardizing the <~1% rule, I made a lot of judgement calls here and the appropriate diff varies widely from test to test.

Reply:
That might be ok to leave out as a judgment call in the docs as well, with some light recommendations. We should always review the threshold/allowed for newly introduced tests, so I think that wouldn't be too high of a risk to mention the approach that was taken when we first introduced it.

Reply:
Added documentation in https://github.com/mapbox/mapbox-gl-js/pull/12268/commits/b9693d2d235c17763d3f3fd6911224d696572e25